konch.py

```diff
@@ -974,6 +980,30 @@ def preview_unauthorized() -> None:
     return None
 
 
+def __ensure_global_cfg_loaded(mod: typing.Union[types.ModuleType, None]) -> None:
+    """Load global config if not already loaded."""
+    global _konch_cfg
+    global _konch_config_registry
+    try:
+        if _konch_cfg is None or _konch_cfg == {} or _konch_cfg == Config():
+            raise EmptyConfigError(f"_konch_cfg is empty: {_konch_cfg}")
```

Comment:
Raising a custom error is unnecessary here if it's just going to get handled below.